From 369845f086fd1e31481aca656cb9131a883a86a9 Mon Sep 17 00:00:00 2001
From: dormando <dormando@rydia.net>
Date: Fri, 24 Apr 2015 00:03:49 -0700
Subject: [PATCH] fix major off by one issue

none of my machines could repro a crash, but it's definitely wrong :/ Very
sad.
---
 memcached.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/memcached.h b/memcached.h
index 689a6f5..99cf0b5 100644
--- a/memcached.h
+++ b/memcached.h
@@ -77,7 +77,7 @@
 
 /* Slab sizing definitions. */
 #define POWER_SMALLEST 1
-#define POWER_LARGEST  255
+#define POWER_LARGEST  256 /* actual cap is 255 */
 #define CHUNK_ALIGN_BYTES 8
 /* slab class max is a 6-bit number, -1. */
 #define MAX_NUMBER_OF_SLAB_CLASSES (63 + 1)
-- 
2.3.6

